packages/web/config/price.ts

```diff
@@ -404,4 +404,14 @@ export const PoolPriceRoutes: IntermediateRoute[] = [
     spotPriceDestDenom: "uosmo",
     destCoinId: "osmosis",
   },
+  {
+    alternativeCoinId: "pool:geo",
+    poolId: "787",
+    spotPriceSourceDenom: DenomHelper.ibcDenom(
+      [{ portId: "transfer", channelId: "channel-258" }],
+      "mgeo"
```

Comment:
the 'g' in 'mgeo' should be capitalized, since the minimal denom is 'mGeo'

Reply:
done